baybe/utils/sampling_algorithms.py

```diff
@@ -41,9 +41,11 @@ def farthest_point_sampling(
         selected_point_indices = [np.random.randint(0, len(points))]
     elif initialization == "farthest":
         idx_1d = np.argmax(dist_matrix)
-        selected_point_indices = list(np.unravel_index(idx_1d, dist_matrix.shape))
+        selected_point_indices = list(
```

Comment:
I think there could actually be a bug in our FPS function! Not related to this PR, but I guess its really time to write a test for it. Can take care of it.

Reply:
Did you already make a work item/discussion for it and can you (potentially there) elaborate on this?

Reply:
Am working on it and will submit a PR with the test soon. But was false alarm, just did read my own docstring properly :D Nonetheless, we should have a test for it anyway (as for pretty much the entire remaining code base 😄), so I'll still do it. Will close here, though